docs/pbc/all/product-information-management/202212.0/install-and-upgrade/install-features/install-the-measurement-units-feature.md

```diff
@@ -1028,7 +1029,7 @@ class CartPageDependencyProvider extends SprykerCartPageDependencyProvider
     protected function getCartPageWidgetPlugins(): array
     {
         return [
-            QuantitySalesUnitWidgetPlugin::class,
```

Comment:
Removed since `spryker-shop/product-measurement-unit-widget:1.0.0`:
* https://github.com/spryker-shop/product-measurement-unit-widget/releases/tag/1.0.0

Reply:
this widget and the one below are now onnected as a global widgets
see `\Pyz\Yves\ShopApplication\ShopApplicationDependencyProvider::getGlobalWidgets`

Reply:
@DmitryLymarenko I've added `ShopApplicationDependencyProvider` instead for both widgets.